sympy/integrals/integrals.py

```diff
@@ -516,6 +517,24 @@ def try_meijerg(function, xab):
                             function = ret
                             continue
 
+            if not isinstance(antideriv,Integral):
+                sym = xab[0]
+                terms = tuple()
```

Comment:
This empty tuple does not seem to be used.

Reply:
yes!!